src/core/scenario.jl

```diff
@@ -577,33 +580,60 @@ function Scenario(d::Dict; flex_hvac_from_json=false)
 
             if hybrid_ghx_sizing_method == "Automatic"
```

Comment:
We can punt on this for now, but let's make it a priority for the GHP task next FY to move all this GHP code to a separate file with a function to load and use in scenario.jl.